package.json

```
@@ -21,7 +21,7 @@
   "scripts": {
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
     "healthcheck": "node ./bin/healthcheck",
-    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
+    "test": "mocha --recursive 'test/unit/entitiesStore_utest.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```

Please take care that local modifications in packages.json to debug locally don't progress to the PR

```suggestion
    "test": "mocha --recursive 'test/unit/**.*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```